cmd/init.go

```diff
@@ -78,6 +78,47 @@ func (m model) View() string {
 	return s.String()
 }
 
+func mergeTemplateConfig(centralConfig *config.Config, templateBruinContent []byte) error {
+	var templateConfig config.Config
+	if err := yaml.Unmarshal(templateBruinContent, &templateConfig); err != nil {
+		return fmt.Errorf("could not parse template's .bruin.yml: %w", err)
+	}
+
+	// Initialize environments map if it doesn't exist
+	if centralConfig.Environments == nil {
+		centralConfig.Environments = make(map[string]config.Environment)
+	}
+
+	// Merge environments and their connections from template into central config
+	for templateEnvName, templateEnv := range templateConfig.Environments {
+		if err := mergeEnvironment(centralConfig, templateEnvName, templateEnv); err != nil {
+			return fmt.Errorf("failed to merge environment %s: %w", templateEnvName, err)
+		}
+	}
+
+	return nil
+}
+
+func mergeEnvironment(centralConfig *config.Config, templateEnvName string, templateEnv config.Environment) error {
```

Comment:
let's say the user has two envs, `staging` and `prod`, `staging` being marked as the `default`. if we have a `.bruin.yml` in our template and that has an env called `default`, it seems like we would create it as a new environment, correct? in that case I suggest using the user's default environment and merge into that.